docs/docs/installation/docker-compose.mdx

```diff
@@ -5,23 +5,29 @@ sidebar_position: 3
 version: 1
 ---
 
-## Using Docker Compose
+import useBaseUrl from "@docusaurus/useBaseUrl";
 
-**DO NOT USE THIS FOR PRODUCTION!**
+# Using Docker Compose
 
-The fastest way to try Superset locally is using Docker Compose on a Linux or Mac OSX
-computer. Superset does not have official support for Windows. It's also the easiest
-way to launch a fully functioning **development environment** quickly.
+
+<img src={useBaseUrl("/img/docker-compose.webp" )} width="150" />
```

Comment:
You don't see webp every day! This won't work with IE11, but that won't keep me up at night.

Reply:
take that IE users! WHAT ARE YOU DOING ON IE?